### PR TITLE
Full volume reflectivity output

### PR DESCRIPTION
**Pull request by anotherjohnwong**
Outputting full 3D volumetric reflectivity.

### CODE REVIEW

**Comment by anotherjohnwong** [comment]
Not quite there yet. Patterned artifacts occur throughout the model domain with values between -35 and zero.

refl10cm

**Comment by bawilt0920** [comment]
Maybe worth a debug attempt and define in loop. Looks ok the way you have it, but maybe something weird with kts:kte?

          do k = kts,kte
             dBZ1d(k) = max(-35._RKIND,dBZ1d(k))
             refl10cm(i,k) = dBZ1d(k)
             if(zp(k) .lt. 1000.) kp = k
          enddo

File: src/core_atmosphere/physics/mpas_atmphys_driver_microphysics.F [hunk 1]
```diff
@@ -598,6 +600,7 @@ subroutine compute_radar_reflectivity(diag_physics,its,ite)
              dBZ1d(k) = max(-35._RKIND,dBZ1d(k))
              if(zp(k) .lt. 1000.) kp = k
           enddo
+          refl10cm(i,kts:kte) = dBZ1d(kts:kte)
```
Comment (bawilt0920) [inline src/core_atmosphere/physics/mpas_atmphys_driver_microphysics.F]: refl10cm(kts:kte,i) = dBZ1d(kts:kte)

Comment (bawilt0920) [reply src/core_atmosphere/physics/mpas_atmphys_driver_microphysics.F]: @anotherjohnwong, Found the issue with refl10cm as mentioned in comment above

**Pull request merged**